xournalpp.appdata.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>xournalpp.desktop</id>
+  <translation type="gettext">xournalpp</translation>
+  <name>Xournal++</name>
+  <summary>Take handwritten notes</summary>
+  <summary xml:lang="ca">Preneu notes a mà</summary>
+  <summary xml:lang="cs">Dělejte si rukou psané poznámky</summary>
+  <summary xml:lang="de">Handnotizen erstellen</summary>
+  <summary xml:lang="fr">Prise de notes manuscrites</summary>
+  <summary xml:lang="pl">Notatki odręczne</summary>
+  <developer_name>Xournalpp Developers</developer_name>
+  <description>
+   <p>
+    Xournal++ is a hand note taking software written in C++ with the target of
+    flexibility, functionality and speed. Stroke recognizer and other parts are
+    based on Xournal Code, which you can find at sourceforge. It supports Linux
+    (e.g. Ubuntu, Debian, Arch, Suse), macOS and Windows 10. Supports pen input
+    from devices such as Wacom Tablets. 
+   </p>
+   <p>Xournal++ features:</p>
+   <ul>
+     <li>Support for Pen preassure, e.g. Wacom Tablet</li>
+     <li>Support for annotating PDFs</li>
+     <li>Fill shape functionality</li>
+     <li>PDF Export (with and without paper style)</li>
+     <li>PNG Export (with and without transparent background)</li>
+     <li>Allow to map different tools / colors etc. to stylus buttons / mouse buttons</li>
+     <li>Sidebar with Page Previews with advanced page sorting, PDF Bookmarks and Layers</li>
+     <li>Enhanced support for image insertion</li>
+     <li>Eraser with multiple configurations</li>
+     <li>Significantly reduced memory usage and code to detect memory leaks compared to Xournal</li>
+     <li>LaTeX support (requires a working LaTeX install)</li>
+     <li>Bug reporting, autosave, and auto backup tools</li>
+     <li>Customizeable toolbar, with multiple configurations, e.g. to optimize toolbar for portrait/landscape</li>
+     <li>Page Template definitions</li>
+     <li>Shape drawing (line, arrow, circle, rect)</li>
+     <li>Shape resizing and rotation</li>
+     <li>Rotation snapping every 45 degrees</li>
+     <li>Rect snapping to grid</li>
+     <li>Audio recording and playback alongside with handwritten notes</li>
+     <li>Multi Language Support, Like English, German (Deutsch), Italian (Italiano)...</li>
+     <li>Plugins using LUA Scripting</li>
+   </ul>
+  </description>
+  <categories>
```

Comment:
If those categories are defined on the desktop file, no need to duplicate them here. Both files are merged with appstream-compose.

Reply:
True, if this is going to be an upstream version we should make sure to review stylistic parts like this on the upstream PR to make sure that we don't get regressions in future updates.

Reply:
yeah, maybe it wasn't too clever of me to open two PRs at the same time. hm.
I've updated both, https://github.com/xournalpp/xournalpp/pull/1358 and this PR to remove the categories as the desktop file does indeed have those categories set.